src/test/java/org/sagebionetworks/bridge/exporter/integration/ExportTest.java

```diff
@@ -0,0 +1,813 @@
+            s3Client.deleteObject(recordIdOverrideBucket, s3FileName);
+        }
+
+        admin.signOut();
+    }
+
+    @AfterClass
+    public static void deleteWorker() throws Exception {
+        if (worker != null) {
+            worker.signOutAndDeleteUser();
+        }
+    }
+
+    @AfterClass
+    public static void deleteDeveloper() throws Exception {
+        if (developer != null) {
+            developer.signOutAndDeleteUser();
+        }
+    }
+
+    @AfterClass
+    public static void deleteResearcher() throws Exception {
+        if (researcher != null) {
+            researcher.signOutAndDeleteUser();
+        }
+    }
+
+    @AfterClass
+    public static void deleteUser() throws Exception {
+        if (user != null) {
+            user.signOutAndDeleteUser();
+        }
+    }
+
+    @Test
+    public void testInstant() throws Exception {
+        DateTime dateTimeBeforeExport = DateTime.now();
+        Long epochBeforeExport = dateTimeBeforeExport.getMillis(); // use for later verification
+        LOG.info("Time before request instant exporting: " + dateTimeBeforeExport.toString());
+
+        TimeUnit.MINUTES.sleep(1); // wait for gray period
+
+        // then execute instant exporting against this new study, done by researcher/developer
+        ForDevelopersApi forDevelopersApi = developer.getClient(ForDevelopersApi.class);
+        Response<Message> response = forDevelopersApi.requestInstantExport().execute();
+        assertEquals(202, response.code());
+
+        // sleep a while to wait for exporter
+        TimeUnit.SECONDS.sleep(20);
+
+        // verify
+        verifyExport(uploadId, 0, false);
+
+        Item lastExportDateTime = ddbExportTimeTable.getItem("studyId", studyId);
+        Long lastExportDateTimeEpoch = lastExportDateTime.getLong("lastExportDateTime");
+        assertNotNull(lastExportDateTimeEpoch);
+
+        // the time received exporting request should be after the time before thread sleep
+        assertTrue(lastExportDateTimeEpoch > epochBeforeExport);
+    }
+
+    @Test
+    public void testDailyOnlyTestStudy() throws Exception {
+        // modify uploadedOn field in record table to a fake datetime earlier than 24 hours ago
+        UploadValidationStatus
+                uploadStatus = user.getClient(ForConsentedUsersApi.class).getUploadStatus(uploadId).execute().body();
+
+        ddbRecordTable.updateItem("id", uploadStatus.getRecord().getId(),
+                new AttributeUpdate("uploadedOn").put(DateTime.now().minusHours(1).minusDays(1).getMillis()));
+
+        assertExport(null, ExportType.DAILY, uploadId, 0, false, false, true);
+
+        // then export another upload
+        createAndEncryptTestFiles(TEST_FILES_A_2, TEST_FILES_B_2);
+        String uploadId2 = testUpload("./legacy-survey-encrypted"); // using the same file name
+
+        // check again
+        assertExport(null, ExportType.DAILY, uploadId2, 0, false, false, false);
```

Comment:
yeah that's my point of using two exports -- the first export will create a last export date time in exportTable, and the test will verify if the second export will use the last export date time instead of exporting all uploads -- since from design part, we assume daily/hourly/instant exporting are actually having same logic: either using the last export date time or, if there is not last export date time, using default time range. I probably needs to modify the first upload record after the first assertExport() in this test to set uploadedOn within 24 hours but before last export date time -- so that it will cover what you talked about.